test/api.js

```diff
@@ -137,6 +137,41 @@ describe('api', function() {
       });
     });
 
+    it('should check SASS_PATH in the specified order', function(done) {
+      var src = read(fixture('include-path/index.scss'), 'utf8');
+      var expected = read(fixture('include-path/expected.css'), 'utf8').trim();
+
+      var separator = process.platform == 'win32' ? ';' : ':';
+      process.env.SASS_PATH = fixture('include-path/functions');
```

Comment:
Could be wrong with the Mocha lifecycle for the `env`, but it's probably good to unset the `SASS_PATH` value in an after test function